app/authentication/authenticator.py

```diff
@@ -61,8 +61,9 @@ def store_session(metadata):
     session.clear()
 
     # get the hashed user id for eq
-    user_id = UserIDGenerator.generate_id(metadata)
-    user_ik = UserIDGenerator.generate_ik(metadata)
+    id_generator = UserIDGenerator(current_app.config['EQ_SERVER_SIDE_STORAGE_USER_ID_ITERATIONS'])
```

Comment:
I think we should pass in `EQ_SERVER_SIDE_STORAGE_USER_ID_SALT` and `EQ_SERVER_SIDE_STORAGE_USER_IK_SALT` into the constructor as they never change during runtime